Lib/enum.py

```diff
@@ -76,8 +77,10 @@ def __setitem__(self, key, value):
                     '_order_', '_create_pseudo_member_',
                     '_generate_next_value_', '_missing_', '_ignore_',
                     ):
-                raise ValueError(f'_sunder_ names, such as "{key}", are '
-                                  'reserved for future Enum use')
+                raise ValueError(
+                        '_sunder_ names, such as %r, are reserved for future Enum use'
```

Comment:
You can use f-string with `!r`.

Reply:
Yes, but I'm not using f-strings anywhere else.

Reply:
At some point I'll switch all the %-interpolations to use f-strings.